src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class RightTriangle extends Default {
+    private double size;
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double size, double firstLeg, double secondLeg) {
+        super(color);
+        this.size = size;
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+
+        return size * size;
+    }
+
+    @Override
+    public double getProperty() {
+        return 0.5 * firstLeg * secondLeg;
```

Comment:
The calculation here is correct for the area of a right triangle, but it should be in the `getArea()` method. Ensure that `getProperty()` is used for a different purpose or rename it.